### ISSUE TITLE
[en][shaman] windfury vs. ghost wolf.

### ISSUE THREAD

**Issue by razgal** [Atlantiss/BugTracker#4914]
## what's wrong:
wind fury procs in ghost wolf
## Proof that it shouldn't :
http://web.archive.org/web/20120712235459/http://www.wowhead.com/spell=8232#comments
"Can't proc in Ghost Wolf (yes, im stupid enough to test that)"

**Comment by deadlycalm** [comment]
you are right everytime but prepare your issues correctly with links to abilities etc. so devs would see that and nerf retarded enh

**Comment by Jenovas** [comment]
deadlycalm - please stop hating

**Comment by xolon** [comment]
Lol.

Ps. Much love for razgal. He's the only one reporting issues like this, keep it up

**Comment by anxanas** [comment]
The link is from TBC aka 2 expansions from Cata, it can easily have changed.

**Comment by Lanrutcon** [comment]
> The link is from TBC aka 2 expansions from Cata, it can easily have changed.
> - anxanas

Wowhead comments are very reliable, it's one of the best sources to get information about something. If the statement is false, it gets downvoted instantly, it it's true then it gets upvoted.

If this was changed in a patch between TBC and Cata, people would commented about it and stopped upvoting the comment. Instead it was upvoted (and no one stated the opposite) until the spell got removed (WoD).
